arm-recoveryservicesbackup/readme.md

```diff
@@ -31,20 +31,24 @@ directive:
 
 ### C#
 
-```yaml
```

Comment:
Some weird merge issue here? Please undo. I think you can undo this with a command like `git checkout upstream/master -- arm-recoveryservicesbackup/readme.ms` .